app-misc/calamares-gentoo-livecd/calamares-gentoo-livecd-3.3.14_p20250915-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="8"
+
+MY_PN="calamares"
+MY_P="${MY_PN}-${PV}"
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit desktop python-single-r1
+
+DESCRIPTION="Gentoo Linux ${MY_PN} installer configuration for LiveCD"
```

Comment:
I wouldn't use ${MY_PN} here.  It's not forbidden per se, but it's ugly and not bringing any value